### ISSUE TITLE
Shader error

### ISSUE THREAD

**Issue by neurolabusc** [Twinklebear/webgpu-volume-raycaster#2]
I realize the WebGPU specification has not been finalized. However, the live demo of this demo fails on Chrome 112.0.5615.50. Other [WebGPU demos](https://webgpu.github.io/webgpu-samples/samples/twoCubes)  work fine on this Windows 10 computer with an old GTX 770 GPU.

````
Tint WGSL reader failure: :2:1 error: unexpected token

type float2 = vec2<f32>;
^^^^

:3:1 error: unexpected token
type float3 = vec3<f32>;
^^^^

:4:1 error: unexpected token
type float4 = vec4<f32>;
^^^^
```

**Comment by Twinklebear** [comment]
Thanks for opening this ticket! You're right, the WGSL code for this repo has fallen out of date, the `type` keyword here has changed to `alias`. I'll update the demo since Chrome is shipping WebGPU in 113

**Comment by Twinklebear** [comment]
Resolved in 2057961f88a2c8cf9d469a0a63e7bb160fec4227 , the website should be updated shortly as well
